packages/firestore/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@firebase/firestore",
+  "version": "0.1.0",
+  "description": "",
+  "scripts": {
+    "dev": "gulp dev",
+    "test": "run-p test:browser test:node",
+    "test:browser": "karma start --single-run",
+    "test:node": "mocha 'test/{,!(integration|browser)/**/}*.test.ts' --compilers ts:ts-node/register -r src/platform_node/node_init.ts --retries 5 --timeout 5000",
+    "prepare": "gulp build"
+  },
+  "main": "dist/cjs/index.node.js",
+  "browser": {
+    "dist/cjs/index.node.js": "./dist/cjs/index.js",
+    "dist/esm/index.js": "./dist/esm/index.js"
+  },
+  "module": "dist/esm/index.js",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@firebase/webchannel-wrapper": "^0.2.2"
+  },
+  "peerDependencies": {
+    "@firebase/app": "^0.1.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.0.4",
+    "@types/mocha": "^2.2.42",
+    "@types/sinon": "^2.3.3",
+    "chai": "^4.1.1",
+    "gulp": "gulpjs/gulp#4.0",
```

Comment:
I think you're missing the grpc dependency.

Reply:
I think you're right! I'll get that added. Great catch